crates/cext/src/transpiler/target.rs

```diff
@@ -484,6 +493,7 @@ impl TargetEntry {
             operation: StandardOperation::Instruction(instruction),
             params: None,
             map: Default::default(),
+            name: Some(instruction.name().to_string()),
```

Comment:
To avoid having to clone the string here, you can just leave this as `None`, let the function ``qk_target_add_instruction`` select whether to use the provided ``name`` or the instruction's name.

```suggestion
            name: None,
```

Reply:
Changed to `None`.